.github/workflows/helm_ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: ci-tests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'charts/**'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint-test-chart:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version:
+          - v1.31.0
+          - v1.30.0
+          - v1.29.0
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.6.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --config .ct.yaml)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: |
+          ct \
+            lint \
+            --config .ct.yaml \
+            ${{github.event_name == 'workflow_dispatch' && '--all'}} ;
+
+      - if: steps.list-changed.outputs.changed == 'true' || github.event_name == 'workflow_dispatch'
+        name: Create kind cluster
```

Comment:
@Koenkk  this is going to signifincantly increase the CI time, nos sure the cost implications in your GH setup. Basically it creates a "dummy"/simplified local kubernetes instance.

Reply:
@tamcore  maybe he is in a paid plan, or he is close to use all his free minutes with the other projects. let's wait for him to confirm it's not an issue.

Reply:
Ah! I thought the 2000 minutes limit would apply here, never used GH fir big projects.